lib/resources/tasks/build.js

```diff
@@ -1,26 +1,40 @@
 import gulp from 'gulp';
+import { CLIOptions, build as buildCLI } from 'aurelia-cli';
 import transpile from './transpile';
 import processMarkup from './process-markup';
 import processCSS from './process-css';
 import copyFiles from './copy-files';
-import {build} from 'aurelia-cli';
+import watch from './watch';
 import project from '../aurelia.json';
 
-export default gulp.series(
+let build = gulp.series(
   readProjectConfiguration,
-  gulp.parallel(
-    transpile,
+  gulp.parallel(transpile,
```

Comment:
sorry for being nitpicky, but could you undo this change (and the one at `copyFiles)`) so that the codestyle matches the rest of the gulp tasks?